gpii/node_modules/flowManager/configs/gpii.flowManager.cloudBased.config.production.json

```diff
@@ -1,5 +1,5 @@
 {
-    "type": "gpii.flowManager.cloudBased.config.development",
+    "type": "gpii.flowManager.cloudBased.config.production",
```

Comment:
Ouch, well spotted.
Yes, there have been enough things touched in this pull, we may as well convert all these configs to json5